applications/decentralized-security-marketplace.md

```diff
@@ -0,0 +1,278 @@
+| **0a.** | License                   | MIT                                                                                                                                                                                                                                                                                      |
+| **0b.** | Documentation             | Inline code documentation for public interface                                                                                                                                                                                                                                           |
+| **0c.** | Testing and Testing Guide | Tests for each public function on the smart contract                                                                                                                                                                                                                                     |
+| **0d.** | Docker                    | N/A                                                                                                                                                                                                                                                                                      |
+|  **1.** | Smart contract API Mock   | Complete interface for both smart contracts without implementation yet: **Pool**: Create project, define types of work inside the bounty, send funds to the rewards pool, create governance space, **Governance**: create proposal, approve proposal, call pool to reward propose author |
+
+### Milestone 2 — Implementation of Pool smart contract
+
+- **Estimated Duration:** 1 month
+- **FTE:**  1.5 
+- **Costs:** 22,000 USD
+
+
+|   Number | Deliverable                 | Specification                                                                                                                                                               |
+|---------:|-----------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|  **0a.** | License                     | MIT                                                                                                                                                                         |
+|  **0b.** | Documentation               | Markdown documents describing user flow and details on how to run RewardsPool contract and call it from the cli                                                             |
+|  **0c.** | Testing and Testing Guide   | N/A                                                                                                                                                                         |
```

Comment:
because intention was to write most of the tests before implementation. the M1 should be very heavy loaded design and initial TDD phase that will give my team understanding what exactly they need to implement and how. 

I expect some additional tests in M2/M3 anyway, so let me add them here. But the main intention that the most coverage will be done in M1